### ISSUE TITLE
[OSX 10.11.2] byte-compile failling on 24.5

### ISSUE THREAD

**Issue by dunn** [Groovy-Emacs-Modes/groovy-emacs-modes#19]
I'm getting the following error(s) when trying to compile `groovy-mode.el`:
```
Eval error in the `c-lang-defconst' for `c-regular-keywords-regexp' in groovy-mode:
Eval error in the `c-lang-defconst' for `c-basic-matchers-before' in groovy-mode:
Eval error in the `c-lang-defconst' for `c-matchers-2' in groovy-mode:

In toplevel form:
groovy-mode.el:354:1:Error: Symbol's function definition is void: set-difference
```
To me it looks like http://debbugs.gnu.org/cgi/bugreport.cgi?bug=18845 wasn't actually fixed in 24.5, since it does build when the `(require 'cl)` is applied to 24.5 in addition to 24.4 and below (https://github.com/Groovy-Emacs-Modes/groovy-emacs-modes/blob/master/groovy-mode.el#L93).

It builds fine with the current `emacs-25` branch though, so no worries there.

**Comment by russel** [comment]
Is this problem still a problem?

**Comment by dunn** [comment]
Nope, looks like this should have been closed with 792b0c5a72f7500c8e07f37b77b96fd1f91ac61b.  Thanks!
